infrastructure/web/routes/admin.js

```diff
@@ -16,6 +16,18 @@ router.get('/orders', getAdminOrders)
 
 router.get('/products', getAdminProducts)
 
+router.get('/products/new', function (req, res) {
```

Comment:
This route is not needed for now. As a start, products and orders will be created only by users and managed by admins